garak/resources/gcg/generate_gcg.py

```diff
@@ -143,7 +146,12 @@ def generate_gcg(
     Returns:
         None
     """
-    mp.set_start_method("spawn")
+    mp.set_start_method("spawn", force=True)
+
+    if target_generator is not None and model_names is not None:
+        msg = "You have specified a list of model names and a target generator. Using the already loaded generator!"
+        print(msg)
```

Comment:
PRINT()?!?!?!?!?!

Reply:
Yeah, this should probably just be logger.warning